### ISSUE TITLE
Structure size bigger than expected when generating c++ code.

### ISSUE THREAD

**Issue by BoozeAddict** [ReClassNET/ReClass.NET#191]
This is a very minor issue, but when the code is generated, the assert part doesn't account for padding. For example , a 0x2C size structure would be padded to 0x30 on an x64 application, to align with 8 bytes. Can be solved by manually changing the assert, or removing it entirely, but dropping this here in case someone gets confused why this happens.

**Comment by KN4CK3R** [comment]
Yes, the assert expects a `#pragma pack(1)` for the classes. There may be a setting to deactivate the assertion in the future.
